packages/alfa-aria/src/get-name.ts

```diff
@@ -539,16 +537,14 @@ function getLabel(element: Element): Option<Element> {
         const root = element.root();
 
         if (root !== element) {
-          const label = root
-            .descendants()
-            .find(
-              and(
-                Element.isElement,
-                (element) =>
-                  element.name === "label" &&
-                  element.attribute("for").some((attr) => attr.value === id)
+          const label = root.descendants().find(
+            and(
+              Element.isElement,
+              and(hasName("label"), (element) =>
+                element.attribute("for").some((attr) => attr.value === id)
```

Comment:
This is of course a candidate for `hasAttribute` but one case is probably not enough reason to lift it from `alfa-rules` to `alfa-dom`…